### PR TITLE
Stgode

### PR DESCRIPTION
**Pull request by Kazeya27**

### CODE REVIEW

**Comment by Kazeya27** [comment]
MAE:2.99
MAPE:0.078
RMSE:5.76

**Comment by Kazeya27** [comment]
{
  "batch_size": 32,
  "train_rate": 0.6,
  "eval_rate": 0.2,
  "input_window": 12,
  "output_window": 12,
  "add_time_in_day": false,
  "add_day_in_week": false,

  "max_epoch": 200,
  "learner": "adam",
  "learning_rate": 0.01,
  "lr_epsilon": 1e-8,
  "weight_decay": 1e-2,

  "lr_decay": true,
  "lr_scheduler": "steplr",
  "step_size": 50,
  "lr_decay_ratio": 0.5,

  "sigma1": 0.1,
  "sigma2": 10,
  "thres1": 0.6,
  "thres2": 0.5

  "evaluator_mode": "average",
}
loss: huber_loss

数据集：PEMSD7(M)
论文结果：
RMSE: 5.66
MAE:   2.97
MAPE: 7.36

libcity结果：
RMSE: 5.76
MAE:   2.99
MAPE: 0.0780
